src/browser/extension/options/MiscellaneousGroup.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export default ({ options, saveOption }) => {
+  const browserName = typeof chrome === 'undefined' ? 'Firefox' : 'Chrome';
+
+  return (
+    <fieldset className="option-group">
+      <legend className="option-group__title">Miscellaneous</legend>
+
+      <div className="option option_value_max-age">
+        <label className="option__label" htmlFor="maxAge">Limit the action history to</label>
+        {' '}
+        <input className="option__element"
+               id="maxAge"
+               type="number"
+               min="1"
+               value={options.maxAge}
+               onChange={(e) => saveOption('maxAge', Number(e.target.value))}/>
+        {' '}
+        <label className="option__label" htmlFor="maxAge">items</label>
+        <div className="option__hint">
+          When the number is reached, DevTools starts removing the oldest actions. Improves the DevTools performance.
```

Comment:
I usually prefer [active voice](http://examples.yourdictionary.com/examples-of-active-and-passive-voice.html) over the passive one because it’s more explicit, and people perceive it easier. So I’d better stick with `DevTools start(s)`. :–)